homeassistant/components/homee/helpers.py

```diff
@@ -0,0 +1,16 @@
+"""Helper functions for the homee custom component."""
+
+import logging
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_name_for_enum(att_class, att_id) -> str:
+    """Return the enum item name for a given integer."""
+    try:
+        attribute_name = att_class(att_id).name
+    except ValueError:
+        _LOGGER.warning("Value %s does not exist in %s", att_id, att_class.__name__)
+        return "Unknown"
+
+    return attribute_name
```

Comment:
I would also argue that this belongs in the library

Reply:
I disagree here, since it is a general function to work with all enums, which have to be passed in, and not connected to homee in in any way.

I would say, if we want this elsewhere, it could fit in HA's helpers

Reply:
I mean if we used this in multiple places in HA it would be a viable helper, but I don't think we do. I kinda see this as an extra tool/util to understand the enums you provide better?

Reply:
I use the name of the DEVICE_PROFILE as model in the device_info, that is where I need that lookup.
The reason is, that Homee does not give any Manufacturer or Model information besides that.